### ISSUE TITLE
Modify page title for patient bulk upload and add patient form

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#5137]
## Area or Component
- /app/patient
- /app/add-patient
- /app/upload-patients

## Quick Summary
The patient bulk upload and add patient form share the same page `<title>`.

## Accessibility Violation
[Page title does not identify purpose of page](https://cdc-devtools.dequecloud.com/rule-help/en/title-not-meaningful)

## Action Requested
Update the `<title>` of the patient bulk upload and patient form so they are unique and identifies the purpose of the page

| path                                                   | new title                                                         |
|--------------------------------------------------------|-------------------------------------------------------------------|
| /app/patient                                           | Edit patient \| SimpleReport                                      |
| /app/add-patient                                       | Add individual patient \| SimpleReport                            |
| /app/upload-patients                                   | Import patients from spreadsheet \| SimpleReport                  |

## Source Code (optional)

[which file or line is the issue attached to?]: #
